core/src/main/java/org/apache/iceberg/hadoop/HadoopTableOperations.java

```diff
@@ -63,15 +64,17 @@
   private final Configuration conf;
   private final Path location;
   private final FileIO fileIO;
+  private final LockManager lockManager;
 
   private volatile TableMetadata currentMetadata = null;
   private volatile Integer version = null;
   private volatile boolean shouldRefresh = true;
 
-  protected HadoopTableOperations(Path location, FileIO fileIO, Configuration conf) {
+  protected HadoopTableOperations(Path location, FileIO fileIO, Configuration conf, LockManager lockMgr) {
     this.conf = conf;
     this.location = location;
     this.fileIO = fileIO;
+    this.lockManager = lockMgr;
```

Comment:
In Iceberg, we avoid abbreviations like `manager -> mgr` because it hurts readability and isn't actually easier to type.